recipe/meta.yaml

```diff
@@ -48,7 +48,6 @@ test:
     - pytest
     - pytest-xdist
     - mock  # [py27]
-  run_test.sh
```

Comment:
Yeah I know. Every time I look at it I want to fix it, but every time I look at it I'm busy doing something else. 😆